client/src/main/scala/org/http4s/client/middleware/Retry.scala

```diff
@@ -61,8 +47,47 @@ object Retry {
   }
 }
 
-
 object RetryPolicy {
+  /** Decomposes a retry policy into components that are typically configured
+    * individually.
+    * 
+    * @param backoff a function of attempts to an optional
+    * FiniteDuration.  Return None to stop retrying, or some
+    * duration after which the request will be retried.  See
+    * `exponentialBackoff` for a useful implementation.
+    * 
+    * @param retriable determines whether the request is retriable
+    * from the request and either the throwable or response that was
+    * returned.  Defaults to `defaultRetriable`.
+    */
+  def apply(
+    backoff: Int => Option[FiniteDuration],
+    retriable: (Request, Either[Throwable, Response]) => Boolean = defaultRetriable
```

Comment:
I started this on release-0.17.x  This should probably have been a `\/`, even though we'll have to change it right back.